packages/nollup/src/index.js

```diff
@@ -0,0 +1,166 @@
+const NAMESPACE = '__PREFRESH__';
+
+module.exports = function() {
+	return {
+		resolveId(id) {
+			if (id === 'prefresh-runtime') {
+				return id;
+			}
+		},
+
+		load(id) {
+			if (id === 'prefresh-runtime') {
+				// TODO: Find way to reuse @prefresh/utils
+				return `    
+                    const isPreactCitizen = name =>
```

Comment:
This should be possible with #28 I'll merge it and publish as 0.1.1 if you think this will suffice